pkg/common/cns-lib/vsphere/virtualcenter.go

```diff
@@ -281,6 +284,13 @@ func (vc *VirtualCenter) connect(ctx context.Context, requestNewSession bool) er
 			return err
 		}
 	}
+	// Recreate VslmClient If created using timed out VC Client
```

Comment:
Shouldnt this be guarded by the FSS?

Reply:
No, this is just a client for Vslm. We will protect the calls made to Vslm endpoint using this client